yw_website/yw_website/urls.py

```diff
@@ -39,12 +39,11 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls,),
-    path('upload/', yw_website_views.model_form_upload, name='upload'),
     path('save/', yw_rest_services_views.create_workflow, name='create'),
     path('save/ping/', yw_rest_services_views.yw_save_ping, name='ping'),
     path('', yw_website_views.home, name='home'),
-    path('my-workflows/', yw_website_views.PersonalWorkflowsView.as_view(), name='my-workflows'),
-    path('detailed_workflow/', yw_website_views.detailed_workflow, name = 'detailed_workflow'),
+    path('my-workflows/', yw_website_views.my_workflows, name='my-workflows'),
+    path('detailed_workflow/<int:document_id>', yw_website_views.detailed_workflow, name = 'detailed_workflow'),
```

Comment:
@aniehuser yes I ended up deciding it might be clearer to have a seperate commit/pr to go and change the variable names. Right now document is used in the html, the views, etc. I do agree they should all be renamed to workflow, or something easier to understand.